art/art.py

```diff
@@ -203,7 +203,7 @@ def randart():
     return art("random")
 
 
-def tprint(text, font=DEFAULT_FONT, chr_ignore=True, decoration=None):
+def tprint(text, font=DEFAULT_FONT, show_font=False, chr_ignore=True, decoration=None):
```

Comment:
1. Add `show_font`  at the end of parameters (after `decoration`)
2. Update docstring